charts/backstage/values.yaml

```diff
@@ -171,14 +171,18 @@ backstage:
   # readinessProbe:
   #   failureThreshold: 3
   #   httpGet:
-  #     path: /healthcheck
```

Comment:
apologies for the late review @nikolaik 🙏
it looks good! Shall we clean up these two outdated references?
https://github.com/backstage/charts/blob/6d92b9fe5dd95bb324b307a9a0d5bef729a12649/charts/backstage/values.yaml#L168

https://github.com/backstage/charts/blob/6d92b9fe5dd95bb324b307a9a0d5bef729a12649/charts/backstage/values.yaml#L188

Reply:
Of course, update the references now! Sorry about the late update, been off work for a while.